catalog/services_state_test.go

```diff
@@ -3,21 +3,23 @@ package catalog
 import (
 	"encoding/json"
 	"fmt"
+	"net/http"
 	"regexp"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/Nitro/sidecar/service"
+	"github.com/jarcoal/httpmock"
```

Comment:
According to [this](https://github.com/jarcoal/httpmock#install), we should be using the new V1 branch of this library (gopkg.in/jarcoal/httpmock.v1).